notebooks/contributing_index.md

```diff
@@ -0,0 +1,40 @@
+Contributing to User Tutorials
+==============================
+
+SimPEG welcomes contributions by the community to improve available tutorial materials.
+However, tutorials must adhere to specific guidelines in order to ensure a high level of quality and consistency.
+For those interested in contributing to SimPEG User Tutorials, please follow the instructions provided.
+
+
+Before Creating a New Tutorial
+------------------------------
+
+```{admonition} WARNING!
+:class: danger
+Please ensure that the contents of your tutorial **DOES NOT** fall into any of the following categories:
+
+* the tutorial focusses on functionality that is not part of SimPEG
+* the amount of SimPEG functionality NOT already covered in another tutorial is insufficient
```

Comment:
Is there a better way to make this statement? It has three negations (the DOES NOT from above) plus the "NOT" and "insufficient". I think it's hard to understand the point.